package br.com.fiap.auth.service.exception

class UserAlreadyExistsException(message: String) : RuntimeException(message)
